pkg/storage/store.go

```diff
@@ -4233,6 +4234,70 @@ func (s *Store) AllocatorDryRun(
 	return collect(), nil
 }
 
+// ManualQueue runs the given replica through the requested queue, returning
+// all trace events collected along the way as well as the error message
+// returned from the queue's process method, if any.
+// Intended to help power an admin debug endpoint.
+func (s *Store) ManualQueue(
+	ctx context.Context, queueName string, repl *Replica, skipShouldQueue bool,
+) ([]tracing.RecordedSpan, string, error) {
+	var queue queueImpl
+	switch strings.ToLower(queueName) {
```

Comment:
Not required, but could this loop through all the queues registered with the scanner and match on the name? Then we'd not have to change anything when adding queues/changing a queue name.